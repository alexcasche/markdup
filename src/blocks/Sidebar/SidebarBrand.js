import styled from 'styled-components'
import { shade } from 'polished'

const SidebarBrand = styled.h1`
  margin: 0;
  color: ${props => shade(0.8, props.theme.colors.grey)};
  font-family: ${props => props.theme.text.fontLogo};
  font-size: 24px;
  letter-spacing: 4px;
  letter-spacing: 4px;
  line-height: 86px;
  text-align: center;
  text-transform: uppercase;
  span {
    color: ${props => props.theme.colors.grey};
  }
`

export default SidebarBrand
