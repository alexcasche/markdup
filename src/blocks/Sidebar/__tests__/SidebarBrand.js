import SidebarBrand from '../SidebarBrand'

describe('SidebarBrand Element', () => {
  it('renders correctly', () => {
    const tree = render(<SidebarBrand />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
