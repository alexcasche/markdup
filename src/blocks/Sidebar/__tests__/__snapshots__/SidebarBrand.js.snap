// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidebarBrand Element renders correctly 1`] = `
.c0 {
  margin: 0;
  color: #888a8f;
  font-family: "Raleway",sans-serif;
  font-size: 24px;
  -webkit-letter-spacing: 4px;
  -moz-letter-spacing: 4px;
  -ms-letter-spacing: 4px;
  letter-spacing: 4px;
  -webkit-letter-spacing: 4px;
  -moz-letter-spacing: 4px;
  -ms-letter-spacing: 4px;
  letter-spacing: 4px;
  line-height: 86px;
  text-align: center;
  text-transform: uppercase;
}

.c0 span {
  color: #ABADB3;
}

<h1
  className="c0"
/>
`;
